modules/oe_webtools_page_feedback/oe_webtools_page_feedback.module

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @file
+ * The Webtools Page Feedback module file.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Cache\CacheableMetadata;
+
+/**
+ * Implements hook_page_attachments().
+ *
+ * Attach smart loader.
+ */
+function oe_webtools_page_feedback_page_attachments(array &$attachments): void {
```

Comment:
Why you attaching this here? Just make add it to the the render array in the block where we render the actual thing...like SocialShareBlock. Which in fact ..you do!

Reply:
Fixed.